go.mod

```diff
@@ -3,10 +3,10 @@ module github.com/hellofresh/health-go/v3
 go 1.14
 
 require (
+	github.com/go-redis/redis/v8 v8.0.0-beta.9
```

Comment:
Maybe we should stick with a stable version? (aka [v7.4.0](https://github.com/go-redis/redis/releases/tag/v7.4.0))

Reply:
makes sense, updated